saleor/graphql/schema.graphql

```diff
@@ -5954,9 +5968,17 @@ type Warehouse implements Node & ObjectWithMetadata {
   shippingZones(before: String, after: String, first: Int, last: Int): ShippingZoneCountableConnection!
   address: Address!
   email: String!
+  isPrivate: Boolean!
```

Comment:
What does exactly describe this flag?

Reply:
`As a User I want to set visibility of warehouse stock to Private so that Customer's won't be able to see the stock at specific warehouse`